doc/spec/estimation/forest.rst

```diff
@@ -363,8 +363,7 @@ and the `ForestLearners Jupyter notebook <https://github.com/microsoft/EconML/bl
     >>> est.fit(Y, T, W, W)
     <econml.ortho_forest.ContinuousTreatmentOrthoForest object at 0x...>
     >>> print(est.effect(W[:2]))
-    [[1. ]
-     [1.2]]
+    [1.00...  1.19...]
```

Comment:
If I understand correctly, the shape changing here is good because it was wrong before.

Should I be concerned that the values themselves have changed as well?

Reply:
Yes, the shape change is intentional. The values have changed because the trees are partitioned into segments ("little bags") that share data, so the same random seed gives slightly different results now.

Reply:
Hm, so the trees are partitioned even when `fit`'s `inference` option is `None` rather than `'blb'`?

Reply:
Yes, otherwise we'd have two ways of partitioning the tress and it would be hard to keep track of. It doesn't affect the estimate much since that's done using all of the `2*n_trees` leaves. The only difference is which data samples a particular tree uses, i.e. groups `slice_len` trees subsample from the same `n_samples/2` samples but which `n_samples/2` are used differs from group to group.